config/initializers/active_storage.rb

```diff
@@ -0,0 +1 @@
+ActiveStorage::Service.url_expires_in = 1.week
```

Comment:
Is this something we can set from the YAML config somehow?

Reply:
1 week is the max - I'd prefer permanent. If we're still experiencing the bug Im seeing now after this, we'll have to do a hack where is the storage service is amazon, we provide the absolute url to the public image on S3/Cloudfront instead of the typical `url_for(logo_file)`.